commands/createchannel.js

```diff
@@ -0,0 +1,21 @@
+require('dotenv').config();
+const { Message } = require('discord.js');
+const createChannel = require('../helpers/create-channel');
+
+/**
+ * 
+ * @param {Message} message 
+ * @param {*} args
+ */
+const run = (message, args) => {
+    const userID1 = '229858084493721600';
+    const userID2 = '638473299072122900';
```

Comment:
Perhaps you could move stuff out of here and pass it in as a param?

Reply:
Ah, this one is there just because the scheduler is not yet done. It will be removed in future.